specs/phase0/weak-subjectivity.md

```diff
@@ -42,7 +42,7 @@ This document uses data structures, constants, functions, and terminology from
 
 ## Weak Subjectivity Checkpoint
 
-Any `Checkpoint` can used be a Weak Subjectivity Checkpoint.
+Any `Checkpoint` can be used as a Weak Subjectivity Checkpoint.
```

Comment:
What do you think about:
```suggestion
Any `Checkpoint` object can be used as a Weak Subjectivity Checkpoint.
```
or just:
```suggestion
Weak Subjectivity Checkpoint is a `Checkpoint` object.
```

/cc @adiasg

Reply:
The first one looks good to me!
> 
> ```suggestion
> 
> Any `Checkpoint` object can be used as a Weak Subjectivity Checkpoint.
> 
> ```
>

Reply:
The first one looks good to me!
```suggestion
Any `Checkpoint` object can be used as a Weak Subjectivity Checkpoint.

```